src/manifest.c

```diff
@@ -812,6 +812,11 @@ static int write_manifest_plain(struct manifest *manifest)
 		}
 
 		string_or_die(&tempmanifest, "%s/Manifest.%s", manifest_tempdir, file->filename);
+		/*
+		 * use_xattr Has to match swupd-client:
+		 * there it is enabled unconditionally in verify_file().
+		 */
+		file->use_xattrs = true;
```

Comment:
@icarus-sparry this sounds like a rather complex solution (these server-side changes plus additional code for this format extension) for no obvious gain. The only advantage that I could come up with for checking xattrs on Manifest files was that signed security.ima provides some additional integrity protection. But that's redundant, swupd itself already does that.